src/app/(proper_react)/redesign/(authenticated)/user/welcome/FindExposures.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import { ProgressBar } from "../../../../../components/client/ProgressBar";
+import styles from "./FindExposures.module.scss";
+
+export const FindExposures = () => {
+  const [scanProgress, setScanProgress] = useState(0);
+  const [scanFinished, setScanFinished] = useState(false);
+  const [checkingScanProgress, setCheckingScanProgress] = useState(false);
+  const router = useRouter();
+
+  const progressSteps = 6;
+  const maxProgress = 100;
+  const totalBreachesCount = 672;
```

Comment:
@rhelmer @mansaj Do you know how we can get the total breaches?

Reply:
`getAllBreaches` from `./src/db/tables/breaches.js` could be used for this, @mansaj is there a better way? I wonder if exposing a function to just return the count would be beneficial.

Reply:
`getAllBreaches` would be an overkill for this (also since this is called every time?!).. we should create a util func in that file to return the count if necessary

If we aren't caching, we should really be caching the number

Reply:
Sorry, the screen I got this from had an error: We actually need to show the total number of data brokers here. Can we get that via API or should we set that in an `.env` for now? Same here: https://github.com/mozilla/blurts-server/blob/main/src/app/(proper_react)/(onboarding)/user/welcome/GetStarted.tsx#L75.

Reply:
Ok, just checked with PM and we’ll show both. Adding that as well.